.rubocop.yml

```diff
@@ -6,6 +6,7 @@ AllCops:
   Exclude:
     - 'vendor/**/*'
     - 'gemfiles/**/*'
+    - 'spec/**/*'
```

Comment:
I think it's reasonable. Let's be careful about formatting and such, but not wasteful.